### PR TITLE
{numlib}[foss/2022b] Armadillo v11.4.3, arpack-ng v3.8.0

### PR DESCRIPTION
**Pull request by jfgrimm**
requires:

- [x] `Eigen`: #16912 
- [x] `Boost`: #17437 

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/4b2bf525db7ab3715a824d127b2166ea for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/a/Armadillo/Armadillo-11.4.3-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name = 'Armadillo'
+version = '11.4.3'
+
+homepage = 'http://arma.sourceforge.net/'
```
Comment (boegel) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-11.4.3-foss-2022b.eb:4]: I don't see a reason to not use `https` here?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3121.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/2672787c9f4b6d179bf39c6850ac0114 for a full test report.

File: easybuild/easyconfigs/a/Armadillo/Armadillo-11.4.3-foss-2022b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/a/Armadillo/Armadillo-11.4.3-foss-2022b.eb]: fixed with [1242e25](https://github.com/easybuilders/easybuild-easyconfigs/pull/17190/commits/1242e25ff7f0d415a9b0aa39542f7f59c8b0566d)

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**